src/common/CqlEditorTerminologySection.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import "./CqlEditorTerminologySection.scss";
+
+interface CqlEditorTerminologySectionProps {
+  title: string;
+  children?: any;
+}
+
+const CqlEditorTerminologySection = (
```

Comment:
can we rename this to `TerminologySection`.  while importing it is `TerminologySection`. also test and css file name

Reply:
Sure, I'll change it